server/instance.go

```diff
@@ -13,9 +13,27 @@ import (
 	"github.com/labstack/echo/v4"
 )
 
+// dbaFlowMiddleware is a feature guard for bb.feature.dba-workflow
+func dbaFlowMiddleware(server *Server, next echo.HandlerFunc) echo.HandlerFunc {
+	return func(c echo.Context) error {
+		role := c.Get(getRoleContextKey()).(api.Role)
+		if role == api.Developer && server.feature(api.FeatureDBAWorkflow) {
```

Comment:
Also needs to check bb.feature.rbac since without rbac, every role is considered Owner

Reply:
Updated. Shall we keep reviewing this PR, or abandon it until we consider the enterprise plan?

Reply:
let's close this for now and revisit later. The instance is actually a pretty tricky part. e.g. There remains product decision what percentage of instance information we need to expose to the developer when dba flow is enabled.